apps/web/vibes/2px/components/radio-button/index.tsx

```diff
@@ -0,0 +1,45 @@
+import clsx from 'clsx'
+
+interface RadioButtonProps extends Omit<React.HTMLProps<HTMLInputElement>, 'type' | 'hidden'> {
+  label: string
+}
+
+const RadioButton = ({ className, label, ...inputProps }: RadioButtonProps) => {
```

Comment:
In another PR comment, Miguel had asked us to standardize our components as `export default function ...` instead of using `const`